modules/installing-oadp-rosa-sts.adoc

```diff
@@ -46,9 +46,11 @@ $ cat <<EOF > ${SCRATCH}/credentials
   [default]
   role_arn = ${ROLE_ARN}
   web_identity_token_file = /var/run/secrets/openshift/serviceaccount/token
+  region=<aws_region>  # <1>
 EOF
 ----
-
++
+<1> <aws_region> is the AWS region in which the cloud resources have been created.
```

Comment:
For this and mobb team's files (cloud-experts)
```suggestion
<1> <aws_region> is the AWS region to use for STS endpoint.
```
could work just as well. this implies not setting this would trigger the OADP-5354 bug.